brayns/parameters/GeometryParameters.cpp

```diff
@@ -119,13 +119,14 @@ GeometryParameters::GeometryParameters()
         "Radius multiplier for spheres, cones and cylinders [float]")(
         PARAM_RADIUS_CORRECTION.c_str(), po::value<float>(),
         "Forces radius of spheres and cylinders to the specified value "
-        "[float]")(
-        PARAM_COLOR_SCHEME.c_str(), po::value<std::string>(),
-        "Color scheme to be applied to the geometry [none|"
-        "neuron_by_id|neuron_by_type|neuron_by_segment_type|"
-        "protein_atoms|protein_chains|protein_residues|protein_backbones]")(
-        PARAM_GEOMETRY_QUALITY.c_str(), po::value<std::string>(),
-        "Geometry rendering quality [low|medium|high]")(
+        "[float]")(PARAM_COLOR_SCHEME.c_str(), po::value<std::string>(),
+                   "Color scheme to be applied to the geometry "
+                   "[none|neuron-by-id|neuron-by-type|neuron-by-segment-type|"
+                   "neuron-by-layer|neuron-by-mtype|neuron-by-etype|neuron-by-"
+                   "target|protein-by-id|protein-atoms|protein-chains|protein-"
+                   "residues]")(PARAM_GEOMETRY_QUALITY.c_str(),
+                                po::value<std::string>(),
+                                "Geometry rendering quality [low|medium|high]")(
```

Comment:
The whole code block for adding options is unreadable. Can you do some hand formatting to all the options and add // clang-format off // clang-format surrounding it?

Reply:
done